docs/src/openapi-v1.yaml

```diff
@@ -643,6 +643,22 @@ paths:
         required: false
         schema:
           type: boolean
+      - name: decodedXcmMsgs
```

Comment:
We need to also add the new fields to the Response for `/blocks`.

Reply:
I added the response by using the [allOf](https://swagger.io/docs/specification/data-models/oneof-anyof-allof-not/#allof) keyword to combine the existing `Block` schema and the newly added `BlockXCM` schema. While I was at it, I did two extra modifications in the docs.